plugins/cisco_meraki_logs.yaml

```
@@ -3,9 +3,11 @@ title: Cisco Meraki
 description: Log parser for Cisco Meraki
 parameters:
   - name: listen_port
+    description: A port which the agent will listen for syslog messages
     type: int
     default: 5140
   - name: listen_ip
+    description: A syslog ip address
```

This makes it sound like this is an IP address of syslog or something; This is actually the local IP to bind to (syslog is the client, we're the server)
```suggestion
    description: The local IP address to listen for syslog connections on
```